ssdeep/ssdeep-lib/ac-fix/Makefile.am

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
There is no git repo. 
The problem is that original makefile do not generate .pc for pkg-config. I contacted author, but for now he didn't response.